python/utils/verify_quote.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+set -e
+
+quote_file_name="quote.bin"
+open_enclave_path=${OPEN_ENCLAVE_PATH:-"/opt/openenclave"}
+
+# Until https://github.com/microsoft/CCF/issues/1468 is done, CCF
+# uses old attestation API to generate quotes
+quote_format="LEGACY_REPORT_REMOTE"
+
+function usage()
+{
+    echo "Usage:"""
+    echo "  $0 https://<node-address> [--mrenclave <mrenclave_hex>] [CURL_OPTIONS]"
+    echo "Verify target node's remote attestation quote."
+    echo "For the verification to be successful, the public key of the node certificate should also match the SGX report data and the corresponding mrenclave should be trusted."
+    echo "If no trusted mrenclave is specified (--mrenclave), the quote measurement should match one of the service's currently accepted code versions."
```

Comment:
I think the `should` here is confusing - it sounds like its describing a user error, but actually describing what the script expects _from the service_.

Perhaps something like:

```suggestion
    echo "Verification involves confirming that the node's certificate matches the SGX report data, and that the mrenclave included in the quote is trusted."
    echo "A specific trusted mrenclave may be provided with --mrenclave. If specified, the quoted mrenclave must match this exactly. If unspecified the service's currently accepted code versions will be retrieved from the target node, and verification will succeed only if the quoted mrenclave is present in this list."
```

Reply:
Good point. I've re-phrased this accordingly (only slight difference from what you suggested).